src/app/InteractionModelEngine.h

```diff
@@ -50,6 +50,9 @@
 namespace chip {
 namespace app {
 
+constexpr size_t kMaxSecureSduLength  = 1024;
+constexpr uint32_t kImMesssageTimeout = 20;
```

Comment:
Please document, especially units.  Also, why the change from 5 to 20?

Reply:
20 is accidental change, would update it back to 5 in next pr